File: 수학(구현)/백준_1105_팔/팔.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int L = s.nextInt();
        int R = s.nextInt();

        int min_cnt = 10;
        for (int i = L; i <= R; i++) {
            int n = i;
            int cnt = 0;
            while (n > 0) {
                if (n % 10 == 8)
                    cnt++;
                n /= 10;
            }
            if (cnt < min_cnt)
                min_cnt = cnt;
            if (min_cnt == 0)
                break;
        }
        System.out.println(min_cnt);

    }
}

File: 수학(구현)/백준_1105_팔/팔.kt
import java.util.*

fun main(args: Array<String>) {
    val s = Scanner(System.`in`)
    val L = s.nextInt()
    val R = s.nextInt()
    var min_cnt = 10

    for (i in L..R) {
        var n = i
        var cnt = 0
        while (n > 0) {
            if (n % 10 == 8) cnt++
            n /= 10
        }
        if (cnt < min_cnt) min_cnt = cnt
        if (min_cnt == 0) break
    }
    print(min_cnt)
}
